MediaBrowser.Model/Dto/PublicUserDto.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using MediaBrowser.Model.Configuration;
+using MediaBrowser.Model.Users;
+
+namespace MediaBrowser.Model.Dto
+{
+    /// <summary>
+    /// Class PublicUserDto. Its goal is to show only public information about a user
+    /// </summary>
+    public class PublicUserDto : IItemDto
+    {
+        /// <summary>
+        /// Gets or sets the name.
+        /// </summary>
+        /// <value>The name.</value>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Gets or sets the primary image tag.
+        /// </summary>
+        /// <value>The primary image tag.</value>
+        public string PrimaryImageTag { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether this instance has password.
+        /// </summary>
+        /// <value><c>true</c> if this instance has password; otherwise, <c>false</c>.</value>
+        public bool HasPassword { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether this instance has configured password.
+        /// </summary>
+        /// <value><c>true</c> if this instance has configured password; otherwise, <c>false</c>.</value>
+        public bool HasConfiguredPassword { get; set; }
```

Comment:
it's unclear to me why this thing would need _two_ "is password enabled" fields

Reply:
Reading this comment in the related issue (https://github.com/jellyfin/jellyfin/issues/880#issuecomment-465370051) it seems that  `HasConfiguredPassword` is necessary if you need to set a password after logging in

Reply:
This is still a stupid thing to have, but (according to my quick search in jellyfin-web) a needed thing to do now.
Please add a `// FIXME` comment here.

Reply:
Done!

Reply:
Actually, I think my findings in jf-web indicate this field is needed whenever you want to change a password, not necessary after a first login. Poetry, man...